jwst/wavecorr/wavecorr.py

```diff
@@ -1,7 +1,30 @@
-#
-#  Module for applying wavelength corrections to NIRSpec MOS and FS
-#  slit instances in which the source is off center.
-#
+"""
+Module for applying wavelength corrections to NIRSpec MOS and FS
+slits in which the source is off center. The correction is applied
+only to point sources.
+
+The algorithm uses a reference file which is a look up table of
+wavelenght_correction as a function of slit_x_position and wavelength.
+
+The slit_x_position is determined in the following way.
```

Comment:
Perhaps it would be useful to mention somewhere near the start that the "x" direction is the one parallel to dispersion/wavelength for both MOS and FS slits.

Reply:
done